documents/2-RTB/Documentazione_Interna/Norme_di_Progetto.typ

```diff
@@ -772,8 +779,91 @@ Per la realizzazione della documentazione sono stati scelti dal team i seguenti
+
+Il team ha adottato un formato specifico per il versionamento dei documenti: *`X.Y.Z`*, dove ciascun valore rappresenta:
+
+- *`X`*: identifica il completamento di una fase significativa del progetto. Questa versione viene definita dal responsabile durante la validazione.
+- *`Y`*: indica una versione intermedia che integra più modifiche incrementali (*`Z`*) in modo coerente. Le versioni con il formato *`X.Y.0`* sono considerate stabili, anche se non completamente definitive.
+- *`Z`*: corrisponde a modifiche incrementali minori, come l’aggiunta o l’aggiornamento di una sezione.
+
+Ogni modifica approvata comporta un incremento di versione, il cui impatto dipende dall’entità della modifica stessa. Inoltre, un aggiornamento di una cifra comporta l’azzeramento delle cifre a destra di essa. Per i verbali e il glossario, viene utilizzato un formato ridotto a due cifre, poiché, per i primi, il testo è generalmente breve e rapido da redigere, per il glossario, le modifiche devono sempre portare a una versione stabile.
+
+=== Tecnologie utilizzate
+- *Git*: strumento per il controllo di versione, utilizzato per tracciare le modifiche a documenti e codice sorgente.
+- *GitHub*: servizio di hosting per progetti software utilizzato per gestire il versionamento e coordinare le attività del team. Include anche un #glossary("Issue Tracking System") per monitorare e gestire segnalazioni e attività.
+
+=== Repository
+==== Lista repository
+Il team all’interno della propria #glossary("GitHub Organization") utilizza 3 repository:
+
+- *`Docs`*: repository destinata alla gestione della documentazione del progetto;
+- *`BuddyBot`*: repository dedicata allo sviluppo e all'implementazione del software;
+- *`SweeTenTeam.github.io`*: repository utilizzata per il sito vetrina del progetto.
+
+==== Struttura della repository Docs
+Il contenuto della repository è organizzato in due #glossary("branch") distinti, ciascuno con una funzione specifica:
+
+- *`master`*: contiene i file PDF compilati.
+- *`develop`*: ospita i sorgenti #glossary("Typst"). Ogni volta che un file viene aggiunto o modificato in questo #glossary("branch"), il sistema avvia automaticamente la compilazione e carica il risultato nel branch `master`, rendendo i documenti compilati accessibili a tutti.
+
+Per entrambe i #glossary("branch"), è stata adottata una struttura chiara che separa i documenti dai file di supporto, come `LICENSE`, `README`, _actions_, _template_ e _images_. Infatti la documentazione è raccolta all’interno della cartella *`documents`*, che include i materiali relativi alle varie fasi del progetto: 
+
+- *`1-Candidatura`*: documenti relativi alla gara per l'aggiudicazione dell'appalto dei capitolati.
+- *`2-RTB`*: _Requirements and Technology Baseline_, definisce i requisiti da soddisfare in collaborazione con il proponente e giustifica la scelta di tecnologie, framework e librerie, dimostrandone l’adeguatezza e la compatibilità.
```

Comment:
la definizione di RTB è giusta ma qua stiamo parlando della cartella ´2-RTB´. Lascia la definizione ma aggiungi cosa è contenuto in questa cartella, tipo "in questa cartella sono presenti tutti i documenti relativi a questa consegna" magari scritto un po' meglio di come ho appena fatto :)

Reply:
done